docs/source/_templates-stable/layout.html

```diff
@@ -0,0 +1,33 @@
+{% extends "!layout.html" %}
+
+{% block menu %}
+
```

Comment:
Is it intentional that we're missing https://github.com/pytorch/pytorch/blob/7eba36b1f60517c478cd708b113317114975e891/docs/source/_templates/layout.html#L3 ? (I don't know what that does)

Reply:
Hmm good question. I'd left it off cause I was following the Sphinx Google Analytics template here: https://www.sphinx-doc.org/en/master/faq.html

Let me see if I can figure out what that line is used for

Reply:
Adding that  canonical url back in to prevent duplicate content issues.
More details about canonical links here: https://yoast.com/rel-canonical/